vllm/worker/worker.py

```diff
@@ -332,9 +348,12 @@ def _check_if_gpu_supports_dtype(torch_dtype: torch.dtype):
 def raise_if_cache_size_invalid(num_gpu_blocks, block_size,
                                 max_model_len) -> None:
     if num_gpu_blocks <= 0:
-        raise ValueError("No available memory for the cache blocks. "
-                         "Try increasing `gpu_memory_utilization` when "
-                         "initializing the engine.")
+        raise ValueError(
+            "No available memory for the cache blocks. vLLM needs {} more GPU "
+            "blocks to allocate. Try increasing `gpu_memory_utilization` when "
+            "initializing the engine. Or increase `tensor_parallel_size`, which"
+            "shards model weights across GPUs. It gives more memory to "
+            "allocate kv cache blocks per GPU.".format(-num_gpu_blocks))
```

Comment:
IIUC, the negative `num_gpu_blocks` does not give any useful information; it just means the memory profiling was inaccurate for some reason.